### PR TITLE
Fix Mobile Styling

### LINKED ISSUE #54: Fix Mobile Styling Issues
Fix a few mobile styling issues on the homepage and state details page:

| | |
| --- | --- |
| Screenshot from 2022-04-19 20-32-29 | Screenshot from 2022-04-19 20-32-40 |
| Screenshot from 2022-04-19 20-33-01 | Screenshot from 2022-04-19 20-32-34 |

### PR DESCRIPTION
**Pull request by vkoves**
## Overview

Fixes mobile issues on the homepage and state details pages by making sure all graphs scale down to mobile properly and scaling a few large pieces of text.

<details>
  <summary>Comparison</summary>

  | Before | After |
  | --- | --- |
  | state-decarbonize netlify app_(iPhone SE) (2) | localhost_8000_(iPhone SE) |

</details>

Closes #54.

### Demo

See description.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Test on mobile, tablet, and desktop
* Also make sure to drag around the width to confirm there's no views at which things look atrocious
* **Note:** Graphs often don't resize properly until refresh, I don't know if it's worth doing a bunch of JS recalculations on resize but if that would be ideal I can toss that in

### CODE REVIEW

**Review by derekeder — CHANGES_REQUESTED** [review]
Looks great! A few issues with the map, though:

- on mobile view for the [deploy preview](https://deploy-preview-60--state-decarbonize.netlify.app/), clicking on a state takes you to `/undefined`. I didn't see any obvious changes that would impact this functionality
- clicking on small states and DC is pretty much impossible on mobile. do we need to list out each state in a list/table in mobile view?

**Comment by vkoves** [comment]
@derekeder - I couldn't reproduce the issue you were having with going to `/undefined` on mobile, but I think you are right on needing a table. Mobile users can also zoom in to click their state (which is what I did for DC) but it feels icky to require that.

**Comment by vkoves** [comment]
As we discussed tonight @derekeder I've added a list of states to improve mobile usability:

Screenshot from 2022-04-26 21-57-36

File: src/pages/index.js [hunk 1]
```diff
@@ -37,6 +37,10 @@ const IndexPage = ({data}) => {
   // Prep data for choropleth map
   const mapData = getLatestEmissions(cleanData)
 
+  const stateSlugs = Object.keys(mapData)
+
+  console.log('stateSlugs', stateSlugs)
```
Comment (derekeder) [inline src/pages/index.js:42]: lets axe this print statement

**Review by derekeder — APPROVED** [review]
looks good! one small change and we can merge this in

**Review by derekeder — APPROVED** [review]

**Pull request merged**